### DISCUSSION TITLE
Environment variable for docker-socket-proxy use

### DISCUSSION THREAD

**Discussion by paralovosca** [louislam/dockge#85] [Feature Request]
Would like a way to utilize an installed docker-socket-proxy.

**Comment by RichyHBM** [comment]
Homepage also allows this if a reference is wanted

**Comment by RichyHBM** [comment]
I think doing this would also allow for not running the image as root

**Comment by tehniemer** [comment]
This is a needed feature.

**Comment by willyp713** [comment]
This is possible using environment variable `DOCKER_HOST` see #357
